src/GovUk.Education.ExploreEducationStatistics.Data.ViewModels/Meta/SubjectMetaViewModel.cs

```diff
@@ -9,4 +11,6 @@ public record SubjectMetaViewModel
     public Dictionary<string, LocationsMetaViewModel> Locations { get; set; } = new();
 
     public TimePeriodsMetaViewModel TimePeriod { get; set; } = new();
+
+    public List<DataSetFileFilterHierarchyViewModel>? FilterHierarchies { get; set; } = null;
```

Comment:
Don't think you need to set an initial value of `null` on a nullable property. Also, since the other properties are auto-instantiated, you should be ok to add `#nullable enable` to this class without raising any warnings/changing anything else.

Reply:
I've removed the `= null;`

There is no need to add `#nullable enable` already set project wide in the csproj file.